src/trusted/service_runtime/sel_main.c

```diff
@@ -569,6 +573,16 @@ int NaClSelLdrMain(int argc, char **argv) {
   }
 
 #if NACL_LINUX
+
+  errno = 0;
+  //if this succeeds, it will copy 0 data, and then we know the LKM is loaded if this if check fails
+  if(process_vm_writev(getpid(), &local_iov, 0, &remote_iov, 0, 32)) {
```

Comment:
I think maybe make this check a function in sel_ldr.c so that we don't clutter main more? Also why did you choose to use puts here?